server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/MailboxGet.scala

```diff
@@ -32,7 +32,9 @@ case class MailboxGetRequest(accountId: AccountId,
                              ids: Option[Ids],
                              properties: Option[Properties])
 
-case class NotFound(value: List[MailboxId])
+case class NotFound(value: List[MailboxId]) {
+  def merge(other: NotFound): NotFound = NotFound(this.value ++ other.value)
+}
 
 case class MailboxGetResponse(accountId: AccountId,
```

Comment:
How about adding a `Found` case classs for `list: List[Mailbox],` so that we mimic what is done for 'NotFound'?

Reply:
Careful on the serialization side though?

Reply:
Why?